app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <title>Turbo Native Directory</title>
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg">
```

Comment:
Are we able to use the Rails [helper](https://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-favicon_link_tag) for this? (we will need to move the svg into `app/assets/images`)

Reply:
Thank you for your feedback!  
I've made the changes as suggested. The SVG is now in app/assets/images, and I used the Rails helper.
Please let me know if there's anything else needed, thank you!